ddl/syncer/state_syncer_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package syncer_test
+
+import (
+	"context"
+	"runtime"
+	"testing"
+	"time"
+
+	. "github.com/pingcap/tidb/ddl"
+	"github.com/pingcap/tidb/ddl/syncer"
+	util2 "github.com/pingcap/tidb/ddl/util"
+	"github.com/pingcap/tidb/infoschema"
+	"github.com/pingcap/tidb/sessionctx/variable"
+	"github.com/pingcap/tidb/store/mockstore"
+	"github.com/pingcap/tidb/util"
+	"github.com/stretchr/testify/require"
+	"go.etcd.io/etcd/tests/v3/integration"
+)
+
+func TestStateSyncerSimple(t *testing.T) {
+	variable.EnableMDL.Store(false)
+	if runtime.GOOS == "windows" {
+		t.Skip("integration.NewClusterV3 will create file contains a colon which is not allowed on Windows")
```

Comment:
Do we now support compiling TiDB on windows?

Reply:
I'm not sure. I think that's supportive. I checked, and the `SecureBootstrap` is not supported.

Reply:
I mean maybe wo could just let it compile failed. But it's not a serious issue. :)